src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -14,6 +14,20 @@ import { replaceDomainsInData } from '../../../utilities/environment/stagingDoma
 
 import MegaMenu from '@department-of-veterans-affairs/formation/MegaMenu';
 
+let defaultLinkData = null;
+
+if (process.env.NODE_ENV !== 'test') {
+  // The MegaMenu data is generated out of the content, rather than a static JSON file.
+  // During the build, the generated link data is written into temporary storage
+  // (project-root/.cache) so that the data is available for import before
+  // Webpack begins compilation. However, the data is not available during Mocha tests
+  // or in any environment that executes this module outside of the build, so we have to
+  // wrap this in a NODE_ENV check.
+
+  // eslint-disable-next-line import/no-unresolved
+  defaultLinkData = require('../../../../../.cache/megamenu.json');
```

Comment:
This is pretty rough, I wonder if we could pass the path or file name into the Webpack config and use a global variable defined there (like we do with __BUILDTYPE__)? That way we would keep some details about how and where the menu data is generated separate from the React code.